### PR TITLE
Updating QUnit to 2.0.1

### PR DESCRIPTION
**Pull request by waratuman**
case 1. Add a new type definition.
- [ ] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [ ] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

Source: http://api.qunitjs.com/category/all/

### CODE REVIEW

File: qunit/qunit-test.ts [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+  var foo = new Foo( 1, "2", [] );
+  var bar: any = {
+    x : 1,
+    y : "2",
+    z : []
+  };
+  assert.propEqual( foo, bar, "Strictly the same properties without comparing objects constructors." );
+});
+
+// QUnit.assert['mod2'] = function( value: any, expected: any, message: string ): any {
```
Comment (waratuman) [inline qunit/qunit-test.ts]: Is there a recommend way to test this? I've left the example commented out because of the no implicit any option.

**Comment by vvakame** [comment]
*qunit/qunit.d.ts*

to author (@diullei). Could you review this PR?
:+1: or :-1:?

Checklist

* [ ] pass the Travis CI test?

**Comment by waratuman** [comment]
Anyway we can move this forward @vvakame? What is the process if the author doesn't respond?

**Comment by vvakame** [comment]
sorry for late response.
I decided to merge this PR.

**Pull request merged**